ismip6_ocean_forcing/model/remap.py

```diff
@@ -245,7 +245,11 @@ def _remap(config, modelFolder):
         dsOut = xarray.open_mfdataset(
             '{}/{}_t_*.nc'.format(progressDir, modelName), combine='nested',
             concat_dim='time')
+        #Fully load netcdf files.
+        dsOut = dsOut.load()
```

Comment:
The documentation doesn't show a return value:
https://docs.xarray.dev/en/stable/generated/xarray.Dataset.load.html
```suggestion
        dsOut.load()
```

Reply:
As I commented here:
https://github.com/ismip/ismip6-antarctic-ocean-forcing/issues/45#issuecomment-2961268803
I don't think this fix should be needed. I think it should work just fine for most people as written and this change might cause other unwanted consequences.

However, since I'm not actively developing this code, I'll be glad to give this a try.